### DISCUSSION TITLE
Proxying Plausible through Svelte / Vercel

### DISCUSSION THREAD

**Discussion by rdupz** [plausible/analytics#1553] [Anything Else]
Hi,
I've subscribed to plausible because of the dns cname feature.
Now it is not supported anymore 😞, but I still need to use plausible without being blocked by browsers...

I saw those [guides](https://plausible.io/docs/proxy/guides/cloudflare) but I'm not able to setup the whole thing using svelte deployed to Vercel.

Can you please provide documentation for this usecase ?

**Comment by metmarkosaric** [comment]
hi! 

cname feature is not effective anymore. you can still use the cname and many of our subscribers still do but proxy is a much better solution. adblockers now crawl the web looking for cnames and can block them when discovered so cname doesn't give you any better stats than our default script. 

adblockers cannot do the same with the proxy. we have several standalone proxy instructions and these can be set up on its own and it does not matter what hosting provider or tech stack you are using. cloudflare is an easy one to start with. see https://plausible.io/docs/proxy/guides/cloudflare

**Reply by metmarkosaric** [reply]
by the way not sure if you saw it but we also have vercel instructions here https://plausible.io/docs/proxy/guides/nextjs

**Comment by erskingardner** [comment]
Is it possible to do a proxy with webflow? I've looked around online but I can't find any details on how to set it up.

**Reply by metmarkosaric** [reply]
i don't think webflow allows you much flexibility in terms of server access etc so cloudflare is probably the easiest one to use with webflow: https://plausible.io/docs/proxy/guides/cloudflare

**Comment by rdupz** [comment]
It was actually quite simple using cloudflare worker. Thanks

**Reply by metmarkosaric** [reply]
you're welcome! and glad to hear that!
